freqtrade/optimize/hyperopt_loss_onlyprofit.py

```diff
@@ -0,0 +1,34 @@
+"""
+OnlyProfitHyperOptLoss
+
+This module defines the alternative HyperOptLoss class which can be used for
+Hyperoptimization.
+"""
+from pandas import DataFrame
+
+from freqtrade.optimize.hyperopt import IHyperOptLoss
+
+
+# This is assumed to be expected avg profit * expected trade count.
+# For example, for 0.35% avg per trade (or 0.0035 as ratio) and 1100 trades,
+# expected max profit = 3.85
+# Check that the reported Σ% values do not exceed this!
+# Note, this is ratio. 3.85 stated above means 385Σ%.
+EXPECTED_MAX_PROFIT = 3.0
```

Comment:
is there a reason to cap profits at 300%? we could also set this to 100 or 10_000?

your formula should also work for higher numbers, and would "stop" optimizing then since it'll return 0 - however if possible i would prefer to optimize to "as high as possible" ... without needing to mess with this here ?